src/com/parse4cn1/command/ParseUploadCommand.java

```diff
@@ -54,6 +54,7 @@ void setUpRequest(ConnectionRequest request) throws ParseException {
 
         if (contentType != null) {
             request.addRequestHeader(ParseConstants.HEADER_CONTENT_TYPE, contentType);
+            addHeader(ParseConstants.HEADER_CONTENT_TYPE, contentType);
```

Comment:
Can you explain why this is an improvement? You're indirectly repeating the line above (see [this line](https://github.com/sidiabale/parse4cn1/blob/master/src/com/parse4cn1/command/ParseCommand.java#L102) in the ParseCommand base class. It effectively repeats what is done in line 56 so I do not see why this is necessary and how it makes any difference.

Please explain.